### PR TITLE
Fallback to flash camera capture in case of insecure origin in Chromium

### PR DESCRIPTION
**Pull request by number213**
In Chromium version 47 [a group of features were disabled](https://sites.google.com/a/chromium.org/dev/Home/chromium-security/deprecating-powerful-features-on-insecure-origins) on insecure origins, including _getUserMedia_.
This fix checks for that case and falls back to flash capture if the origin is insecure.

### CODE REVIEW

File: dist/FileAPI.html5.js [hunk 1]
```diff
@@ -107,6 +107,7 @@
 		userAgent = window.navigator.userAgent,
 		safari = /safari\//i.test(userAgent) && !/chrome\//i.test(userAgent),
 		iemobile = /iemobile\//i.test(userAgent),
+		insecureChrome = /chrome\//i.test(userAgent) && window.location.protocol === 'http:',
```
Comment (RubaXa) [inline dist/FileAPI.html5.js]: then + `!safari && `

Comment (number213) [inline dist/FileAPI.html5.js]: Line 108 (`safari`) checks for Chrome userAgent anyway, so if `/chrome\//i` passes it's the same as `!safari`, isn't it?

Comment (RubaXa) [inline dist/FileAPI.html5.js]: If into Safari is exists the same problem, then OK, otherwise need check `!safari`.

**Comment by number213** [comment]
Added Safari check

**Pull request merged**